.github/actions/e2e/action.yaml

```diff
@@ -161,3 +161,23 @@ runs:
     with:
       name: logs-${{ inputs.spark_version }}-${{ inputs.scala_version }}
       path: "*.log"
+  - name: Confirm # of executors
+    if: always()
+    run: |
+      # Confirm # of executors
+      echo "::group::confirmExecutors"
+      expectedExecutorCount=`grep 'conf spark.executor.instances' ./armada-spark/scripts/submitSparkPi.sh | grep -oP 'spark\.executor\.instances=\K\d+'`
+      actualExecutorCount=`grep "ArmadaDriverEndpoint: Registered executor" armada-${{ steps.submit.outputs.jobid }}-0.log | wc`
+      echo executor count expected: $expectedExecutorCount actual: $actualExecutorCount
+      if [ "$expectedExecutorCount" = "$actualExecutorCount" ]; then
```

Comment:
Should this comparison be done by `-eq` (integer comparison) instead of `=` (string comparison)?

Reply:
(Of course, this means removing the quotes around `$expectedExecutorCount`, etc, so if it feels like pulling a string on a sweater, this will probably be fine anyway. Bash is annoying at times.)

Reply:
i'll fix it, thanks!

Reply:
👏🏼